pkg/grub/Dockerfile

```diff
@@ -28,7 +30,7 @@ search_disk_uuid search_part_label search_label xzio xfs video gfxterm serial gp
 part_msdos cat echo test configfile loopback net tftp http true"
 
 FROM grub-build-base AS grub-build-amd64
-ENV GRUB_MODULES="multiboot multiboot2 efi_uga efi_gop linuxefi gpt verify gcry_sha256 measurefs efinet"
+ENV GRUB_MODULES="multiboot multiboot2 efi_uga efi_gop linuxefi gpt verify gcry_sha256 measurefs efinet getenv"
```

Comment:
And for ARM?

Reply:
@OhmSpectator 
1. it is also for ARM. (forgot to fix commit description)
2. end users should not use it imo
3. script is just for testing, I did not make it erganomic.  we can add one later